SpatialGDK/Source/SpatialGDK/Public/EngineClasses/AbstractVirtualWorkerTranslator.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+
+class SPATIALGDK_API AbstractVirtualWorkerTranslator
```

Comment:
This class doesn't seem to really be an abstract translator; that would have all the input functions etc too. It's really just a local ID provider.

Reply:
I think we'll want to expand this set of functions when other classes have tests made and require more functionality out of the virtual worker translator mock. Once that happens we'd likely just to rename back to what it is now, or at least something broader than local ID provider so I'd rather leave it like this?

Reply:
You still aren't describing an abstract translator. That's still a hack to make tests. If you want to name it like this then I think this should actually be made into an abstract translator.

Reply:
If the test hack happens to coincide with an abstract translator at some later date it's still a test hack. To change that I need to know that I can replace a translator with any instance of the abstract translator and know everything will work.